src/main/java/net/sf/jabref/gui/FieldSetComponent.java

```diff
@@ -364,4 +369,29 @@ public void actionPerformed(ActionEvent e) {
         }
     }
 
+    /**
+     * FocusListener to select the first entry in the list of fields when they are focused
+     */
+    protected class FieldListFocusListener implements FocusListener {
+
+        private JList list;
+
+        public FieldListFocusListener(JList list) {
+            this.list = list;
+        }
+
+        @Override
+        public void focusGained(FocusEvent e) {
+            if (list.getSelectedValue() == null) {
+                list.setSelectedIndex(0);
```

Comment:
The clicked item is selected since the selection value is no longer null. I tested it to be sure.